libs/api-dh-app/source/DataHub.WebApi/WeatherForecast.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Energinet DataHub A/S
```

Comment:
Not sure why all these lines are changed, but the CI changes looks good to me :)

Reply:
They where changed to "UTF-8 with BOM" :)